php8.3-dev/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM thebiggive/php:8.3
```

Comment:
this file is a copy of https://github.com/thebiggive/docker-php/blob/b8e9c544addee6716c42a99d6e27368a11638f8c/Dockerfile#L11-L10 , just changed 8.1 to 8.3